_includes/head.html

```diff
@@ -1,4 +1,5 @@
 <head>
+    <link rel="shortcut icon" href="">
```

Comment:
pls add a favicon similar to the one used at other summit sites

Reply:
There is no need i think

Reply:
You mean `<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>` this right?

Reply:
yes, pls we need an icon